spec/lib/audit/catalog_to_moab_instance_spec.rb

```diff
@@ -125,11 +128,53 @@
         allow(PreservedObjectHandlerResults).to receive(:new).and_return(pohandler_results)
         c2m.check_catalog_version
       end
-      it 'does moab validation' do
-        skip 'add tests after #491'
+
+      it 'calls Stanford::StorageObjectValidator.validation_errors for moab' do
+        mock_sov = instance_double(Stanford::StorageObjectValidator)
+        expect(mock_sov).to receive(:validation_errors).and_return([])
+        allow(Stanford::StorageObjectValidator).to receive(:new).and_return(mock_sov)
+        c2m.check_catalog_version
+      end
+      it 'valid moab sets status to EXPECTED_VERS_NOT_FOUND_ON_STORAGE_STATUS' do
+        orig = pres_copy.status
+        c2m.check_catalog_version
+        new_status = pres_copy.reload.status
+        skip 'uncomment rest of tests after #478 - we need to save the object'
+        expect(new_status).not_to eq orig
+        expect(new_status).to eq PreservedCopy::EXPECTED_VERS_NOT_FOUND_ON_STORAGE_STATUS
+      end
+      context 'invalid moab' do
+        before do
+          mock_sov = instance_double(Stanford::StorageObjectValidator)
+          allow(mock_sov).to receive(:validation_errors).and_return([foo: 'error message'])
+          allow(Stanford::StorageObjectValidator).to receive(:new).and_return(mock_sov)
+        end
+        it 'sets status to INVALID_MOAB_STATUS' do
+          orig = pres_copy.status
+          c2m.check_catalog_version
+          new_status = pres_copy.reload.status
+          skip 'uncomment rest of tests after #478 - we need to save the object'
+          expect(new_status).not_to eq orig
+          expect(new_status).to eq PreservedCopy::INVALID_MOAB_STATUS
+        end
+        it 'adds an INVALID_MOAB result' do
+          pohandler_results = instance_double(PreservedObjectHandlerResults, report_results: nil)
+          expect(pohandler_results).to receive(:add_result).with(
+            PreservedObjectHandlerResults::INVALID_MOAB, anything
```

Comment:
I was expecting to find `anything` defined, but it isn't. Is that because of `any_args` in the next line?

Reply:
`anything` is actually a reserved work for rspec and it means "any single argument", while `any_args` is any number of anything arguments.